src/go/rpk/pkg/cli/cmd/debug/bundle/bundle_k8s_linux.go

```diff
@@ -15,13 +15,18 @@ import (
 	"archive/zip"
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
+	"regexp"
+	"strings"
+	"time"
 
 	"github.com/hashicorp/go-multierror"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
 	"github.com/spf13/afero"
+	"github.com/trstringer/go-systemd-time/pkg/systemdtime"
 	v1 "k8s.io/api/core/v1"
```

Comment:
Nit: this would probably be better named as `k8score` or something like that -- in usage below, it's difficult to know what package `v1` is.